src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java

```diff
@@ -20,6 +20,7 @@ public interface TombsOfAmascutConfig extends Config
 {
 
 	String CONFIG_GROUP = "tombsofamascut";
+	String CONFIG_KEY_DRYSTREAK = "drystreak";
```

Comment:
Replace this with a hidden config entry + setter ([example](https://github.com/LlemonDuck/tombs-of-amascut/blob/master/src/main/java/com/duckblade/osrs/toa/TombsOfAmascutConfig.java#L635-L652))